pkg/vpcmodel/grouping.go

```diff
@@ -20,8 +20,11 @@ type groupedNodesInfo struct {
 }
 
 type explainDetails struct {
-	rules  *rulesConnection
-	router RoutingResource
+	rules          *rulesConnection
+	router         RoutingResource
+	connEnabled    bool
```

Comment:
seems this field is not used (`connEnabled`)

Reply:
It is indeed not used at the moment. Lets keep it until we finalize all the printing formats. If it is still not used then, I'll remove it. I'll add a todo for now.